spec/index.html

```diff
@@ -369,11 +369,13 @@ <h3>Terms defined by this specification</h3>
         of nodes and quads such that the first quad includes
         the starting node as an component, and the last quad includes
         the ending node as an component with each quad in the path
-        containing both the preceding and following nodes.</dd>
+        containing both the preceding and following nodes. Gossip paths are
+        encoded and operated on in the <a>RDFC-1.0</a> algorithm as
+        strings.</dd>
```

Comment:
Nothing here *defines* `gossip path`, and it is not clear whether such a thing is only found between two `blank [i.e., unnamed] nodes`, or also exists between `named nodes`.

Reply:
I've added what I hope is a better definition of gossip paths in b8f9a52a3056977faa6fd232af829ee2cc6cf8cc. Let me know if this addresses your concern.

Reply:
Another minor fix here: 3eb75deda3c0d77404c811c6bebb8239a3078818.

Reply:
Tried to refine the language once more into multiple sentences here: 0d559e0aa98bc90e6dc9407a6c9449cf099bb669